docs/standard/security/fips-compliance.md

```diff
@@ -0,0 +1,30 @@
+---
+title: .NET Core FIPS compliance
+description: Explains .NET Core FIPS compliance
+ms.date: 11/20/2019
+author: "Rick-Anderson"
+ms.author: "riande"
+---
+
+# .NET Core FIPS compliance
+
+.NET Core:
+
+* Does **not** implement any cryptographic primitives.
```

Comment:
This depends on what "any" and "primitives" mean.

We have an implementation of Password-Based Key Derivation Functions 1 and 2 (PBKDF1, PBKDF2).  Are those "primitives"?  Or Password-Based Encryption Schemes 1 and 2 (PBES1, PBES2)?

Reply:
Does **not** implement **MOST** ??

Reply:
With a few exceptions,  does **not** implement cryptographic primitives.

I could put a footnote and list what you provided if you 'd prefer a footnote.

Reply:
Yeah, I'm not quite sure how we would phrase this here, but we can explicitly say that we use system implementations for AES, DES, ECDH, ECDSA, HMAC-MD5, HMAC-SHA1, HMAC-SHA256, HMAC-SHA384, HMAC-SHA512, MD5, RC2, RSA, SHA-1, SHA-256, SHA-384, SHA-512, or TripleDES (aka 3DES, aka DES-EDE).

We do have implementations of algorithms and schemes that use these cryptographic primitives, such as PBKDF1, PBKDF2, PBES1, PBES2, and HKDF.

Reply:
@bartonjs  we could delete the line
* Does **not** implement any cryptographic primitives.